flocker_bb/steps.py

```diff
@@ -233,16 +255,19 @@ def _merge(self, date):
             'GIT_AUTHOR_DATE': date.strip(),
             'GIT_COMMITTER_DATE': date.strip(),
         })
-        return self._dovccmd(['merge',
-                              '--no-ff', '--no-stat',
-                              'FETCH_HEAD'])
+        merge_target = self.getProperty('merge_target', 'FETCH_HEAD')
```

Comment:
So this is the value of `lint_revision`, which we set earlier. 
Under what circumstances will that property *not* be set and will we use FETCH_HEAD instead?
Maybe add an explanatory comment here.
Otherwise,  I think I get it.